cli/src/main/java/com/devonfw/tools/ide/environment/AbstractEnvironmentVariables.java

```diff
@@ -140,7 +140,7 @@ protected void collectVariables(Set<String> variables) {
 
   /**
    * @param propertiesFilePath the {@link #getPropertiesFilePath() propertiesFilePath} of the child
-   *        {@link EnvironmentVariables}.
```

Comment:
I am note sure what the correct code style is. I think I loaded the `settings/eclipse/ ide-code-style.xml` corretly. So does this style comply with the agreed upon?